### ISSUE TITLE
Fake subdomain links

### ISSUE THREAD

**Issue by jangermuller** [lblissett/dn-discuss#16]
Hi, I've now collected the following subdomains:

This already works - great!
http://dnc3aled.discourseanalysis.net | https://discourseanalysis.net/DNC3ALED/EN

We've created these new pages and they need to be linked with the subdomains:
http://dnc3aled.diskursanalyse.net | https://discourseanalysis.net/DNC3ALED/DE
http://dnc3aled.analysedudiscours.net | https://discourseanalysis.net/DNC3ALED/FR
http://dnc3aled.analisisdeldiscurso.net | https://discourseanalysis.net/DNC3ALED/SP
http://dnc3aled.analisedodiscurso.net | https://discourseanalysis.net/DNC3ALED/DE
http://publications.discourseanalysis.net | https://discourseanalysis.net/DN/publications
http://dncwps.discourseanalysis.net | https://discourseanalysis.net/DN/publications/dnwps
http://discoursenet.discourseanalysis.net | https://discourseanalysis.net/DN
http://dn22.discourseanalysis.net | http://discourseanalysis.net/DN/DN22
http://dn23.discourseanalysis.net | https://discourseanalysis.net/DN/DN23

Die folgenden Subdomains muessen wohl noch geschaffen warden. Wie geht das? |  
Create: Semlangsoc.analysedudiscours.net | https://discourseanalysis.net/SEMLANGSOC
Create: Dipe.discourseanalysis.net | https://discourseanalysis.net/DIPE/
Create: dipe4.discourseanalysis.net | https://discourseanalysis.net/DIPE/DIPE4
Delete: psd.discourseanalysis.net |

**Comment by bits-of-static** [comment]
Subdomains would be connected when the pages are there.

In case they do not exist yet, they have to be created. This is currently not under my supervision. We should consider freeing Daniel from this part of the workflow.

Why do you want to create (so many) subdomains? The URLs are of equivalent length and they do not provide any advantage and do not justify their usage. The overhead of creating and later administering them puts unnecessary weight on the system. Also, this design will most probably also be down-rated by search engines.

Another point against: Once you open them, there is no nice way of getting rid of them, when they loose relevance.

**Comment by bits-of-static** [comment]
We are also missing those:
```
#REDIRECT: DiscourseNet
discoursenet.discourseanalysis.net

#REDIRECT: DISCONEX
disconex.discourseanalysis.net

#REDIRECT: PSD Palgrave Postdisciplinary Studies in Discours
psd.discourseanalysis.net

#REDIRECT: DNS2 Spring School Number 2 Pamplona
dns2.discourseanalysis.net

#REDIRECT: Collaborative Working Paper Series
dncwps.discourseanalysis.net

#REDIRECT: Palgrave Discourse Publications
publications.discourseanalysis.net

#REDIRECT: Symposium DAVNL
davnl.discoursanalyse.net

#REDIRECT: Imagining the people (IMP)
populism.discourseanalysis.net

#REDIRECT: Science Knowledge Valuation workshop
sciencews.discourseanalysis.net
```

**Comment by jangermuller** [comment]
The pages should be there (NB: I've corrected two minor mistakes in the paths above). I find subdomains are easier to remember. I have the domain accounts and would be happy to create them (if I knew how). In your list we would discontinue everything except

#REDIRECT: DiscourseNet
discoursenet.discourseanalysis.net

#REDIRECT: DISCONEX
disconex.discourseanalysis.net

#REDIRECT: Collaborative Working Paper Series
dncwps.discourseanalysis.net

#REDIRECT: Palgrave Discourse Publications
publications.discourseanalysis.net

But it's true the new platform is much more intuitive and we could do with few of them, especially if it's bad for the search engines. Jan, Jaspal what would you think?

**Comment by jangermuller** [comment]
After reflection, I think we need these subdomains since they have been communicated to many people! But we don't have to be generous with future ones. 

NB: this is a real subdomain
#REDIRECT: DISCONEX
disconex.discourseanalysis.net

**Comment by jangermuller** [comment]
So if you could now link them accordingly? Thanks!

**Comment by jangermuller** [comment]
http://dnc3aled.discourseanalysis.net should refer to https://discourseanalysis.net/dnc3-aled-congressbut it sometimes leads me to https://discourseanalysis.net/DNC3ALED/EN! Is this my browser or is there an error?

**Comment by jangermuller** [comment]
Yes, http://dnc3aled.discourseanalysis.net now leads me to https://discourseanalysis.net/en/third-discoursenet-aled-congress-dnc3-aled-knowledge-and-power-polycentric-world-discourses If we could solve the subdomain issue as fast as possible? People will not be able to register!

**Comment by bits-of-static** [comment]
Yes, those were temporary issues resulting from browser cache, DNS network inconsistencies, etc.

See the list at the top of the issue. Let me know once the pages are there, I will link in the rest of subdomains.

**Comment by jangermuller** [comment]
Ich schreib mal, was heute sehr dringend ist: 

1) DNC3ALED am Anfang auf den Events der Landing Page und 

2) die vier Fake Subdomains der Konferenz, die immer noch nicht gehen, naemlich 
 http://dnc3aled.diskursanalyse.net | https://discourseanalysis.net/DNC3ALED/DE
  http://dnc3aled.analysedudiscours.net | https://discourseanalysis.net/DNC3ALED/FR
  http://dnc3aled.analisisdeldiscurso.net | https://discourseanalysis.net/DNC3ALED/ES
  http://dnc3aled.analisedodiscurso.net | https://discourseanalysis.net/DNC3ALED/PT 

3) Die naechste Prioritaet danach waere die Moeglichkeit, Nachrichten zu versenden.

**Comment by bits-of-static** [comment]
2. Weiß nicht warum die Änderungen noch nicht gegriffen haben - geht jetzt.

**Comment by jangermuller** [comment]
Bei mir nicht!!

**Comment by bits-of-static** [comment]
Browser cache. Lass wissen wenn die restlichen Seiten mit Inhalt befüllt sind. Bis dahin zeigen die domains auf die jeweiligen events.

**Comment by jangermuller** [comment]
Safari geht! Gut.
